cookbooks/bcpc-hadoop/recipes/oozie_pack.rb

```diff
@@ -0,0 +1,98 @@
+#
+# Recipe to include hbase jar files into Oozie
+#
+%w{
+hbase
```

Comment:
Would recommend to use the HBase jars shipped with the version of HDP being installed on the cluster.